### ISSUE TITLE
3.0beta relies on openssl1.1.0 feature (help)

### ISSUE THREAD

**Issue by afosgerau** [drwetter/testssl.sh#1068]
3.0beta apparently tries calling openssl s_client with option -help. It seems like -help was first introduced in openssl 1.1.0. At least my openssl 1.0.2-chacha on CentOS 6.9 doesn’t like it.

**Comment by drwetter** [comment]
Nah, that's just a trick to get the s_client features printed out on stderr (and catch those for further processing). 

On CentOS 7.0 (just tested again) and every other distro I saw so far it works fine. Could you catch the output and post it here?

**Comment by afosgerau** [comment]
Silly me. An incorrect parameter led me to use debug mode and I was confused by the debug output in s_client_has.txt. Thank you for your prompt response and a wonderful tool. Hope that either I or someone else will succeed in getting it integrated with Nagios and/or other monitoring tools.
